vue/components/ui/atoms/input/input.vue

```diff
@@ -11,6 +11,7 @@
         v-on:input="onInput($event.target.value)"
         v-on:focus="onFocus"
         v-on:blur="onBlur"
+        v-on:keyup.enter="onEnter"

```

Comment:
Let's forward the whole `keyup` instead, it's a little more flexible: `v-on:keyup=stuff / stuff(event) { $emit('keyup', event)`